src/stories/form/StyledBankAccountListItem.tsx

```diff
@@ -1,21 +1,33 @@
 import { BankAccount } from '../../api/definitions/bank-account';
 import { Utils } from '../../utils';
-import DfxIcon, { IconVariant, IconColors } from '../DfxIcon';
+import DfxIcon, { IconVariant, IconColors, IconSizes } from '../DfxIcon';
+import StyledIconButton from '../StyledIconButton';
 
 export interface StyledBankAccountListItemProps {
   bankAccount: BankAccount;
+  onDelete: () => void;
 }
 
-export default function StyledBankAccountListItem({ bankAccount }: StyledBankAccountListItemProps) {
+export default function StyledBankAccountListItem({ bankAccount, onDelete }: StyledBankAccountListItemProps) {
   const isLabelAvailable = bankAccount.label && bankAccount.label.length > 0;
   return (
-    <div className="flex gap-3 mb-3 last:mb-0 rounded py-1 px-3 hover:bg-dfxGray-400/50 ">
-      <div className="place-self-center">
-        <DfxIcon icon={IconVariant.BANK} color={IconColors.BLACK} />
+    <div className="flex  mb-3 last:mb-0 group justify-between rounded py-1 px-3 hover:bg-dfxGray-400/50 ">
+      <div className="flex gap-3">
+        <div className="place-self-center">
+          <DfxIcon icon={IconVariant.BANK} color={IconColors.BLACK} />
+        </div>
+        <div className="flex flex-col justify-center h-[42px]">
+          {isLabelAvailable && <p className="text-dfxGray-800 text-xs">{bankAccount.label}</p>}
+          <p className="text-dfxBlue-800">{Utils.formatIban(bankAccount.iban)}</p>
+        </div>
       </div>
-      <div className="flex flex-col justify-center h-[42px]">
-        {isLabelAvailable && <p className="text-dfxGray-800 text-xs">{bankAccount.label}</p>}
-        <p className="text-dfxBlue-800">{Utils.formatIban(bankAccount.iban)}</p>
+      <div className="hidden group-hover:block">
+        <StyledIconButton
+          icon={IconVariant.DELETE_OUTLINE}
+          color={IconColors.RED}
+          size={IconSizes.LG}
+          onClick={onDelete}
+        />
```

Comment:
We will need to deactivate / hide this button for now.
API is ready, but exchange hasn't implemented the functionality for it yet.
If we need to release it early due to asset icon changes, we need to hide it for now

Reply:
yes, i thought about it this morning, that i wanted to discuss with you two. if today is just about the token, i should comment that stuff out ...